### PR TITLE
Ensuring logits scores are contiguous resolved the bug on MPS，Mac MPS bug修复

### PR DESCRIPTION
**Pull request by LeoN0425**
发现代码在Mac的MPS框架上运行时，效果很差，这是因为CustomRepetitionPenaltyLogitsProcessorRepeat的处理在MPS上，可能会发生score变量数值错误的未知异常，将其连续化scores.contiguous(), 可以解决此问题。

### CODE REVIEW

**Review by fumiama — APPROVED** [review]
Thanks!
